app/data_model/session_data.py

```diff
@@ -14,7 +14,7 @@ def __init__(self,
                  account_service_url=None,
                  submitted_time=None,
                  trad_as=None,
-                 **_):
+                 **_):                  # pylint: disable=too-many-locals
```

Comment:
Curious question...are these all required for session data?

Reply:
They all look as if they are useful. But I have not tracked the usage of all the params .